src/components/Input.tsx

```diff
@@ -28,19 +57,20 @@ const InputMsg = (props: InputProps) => {
         name="msg"
         placeholder="Send a message .."
         className={styles.input}
-        value={msg}
+        value={msg || transcript}
```

Comment:
as we combined the message with the transcript above here we can read from the msg value only